src/main/java/org/moskito/control/ui/action/inspection/ShowAccumulatorsListAction.java

```diff
@@ -34,8 +34,13 @@ public ActionCommand execute(ActionMapping mapping, FormBean formBean, HttpServl
         if (application == null) {
             return mapping.error();
         }
```

Comment:
getComponent(String) method never return null. It throws IllegalArgumentException instead